src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/service/AuditServiceTest.kt

```diff
@@ -197,4 +200,94 @@ class AuditServiceTest {
       }
     }
   }
+
+  @Nested
+  inner class findFilteredAuditEvents {
+
+    @Test
```

Comment:
Sorry Ola, think you were just following the pattern here. Can discuss this with Karen but to my mind this unit test could be made bit more succinct. Not convinced that we need a list of 4 events to verify the code. Also think the test should verify that all expected parameters are correctly passed on to the repository.

Reply:
That is fine. 
We need to appreciate the audit service is WIP and some of the existing functionality is almost POC like, which reflects in some of the patterns used both in the test and CUT. 
The ask was to add some functionality as per the ticket and improve the entire api where deemed fit, regardless of who was the original authors.
So, we are on track and good team work!